Create decisions.md to document permissions

meganhkelley opened the pull request:
It was recently decided that we should give VFS teams write access to this repo. We still expect them to follow content review procedures with VSP, but are not (currently) enforcing that via a technical block at the point of code review. This file documents that decision and who endorsed it.

## Page to edit
url: 

## Origin of request (internal/stakeholder/user feedback)


## Description of what's needed (edits/link changes/additions)

ncksllvn commented on decisions.md:
```diff
@@ -0,0 +1,5 @@
+
+|Date | Decision | Endorsed By|
+|-- | -- | -- |
+|11/20/19 | VFS teams can have write access to this repository, which allows them to commit changes to master with only 1 code review, which can be a peer. Teams are still expected to submit content to platform (VSP) for review before publishing. This decision simply means that we are not enforcing that content review process with a technical block in the form of required code review from a certa ...
```
Feel free to paste this right in the README if that'll increase visibility. Also, a decision title or shorthand description might make it a little easier to catch the gist.

meganhkelley replied:
Ah yep good catch will do, thanks!

meganhkelley commented:
Updated decisions file: added bold decision title as summary, moved file to /docs folder.
Updated README with summary of repo permissions, with link to VSP content ROE and to the new decisions file.

ncksllvn commented:
Sounds great @meganhkelley
